source/get-started/pojo-qs.txt

```diff
@@ -0,0 +1,410 @@
+``updateOne()`` and ``updateMany()`` methods.
+
+Pass the following components to the methods:
+
+- Filter object to determine the document or documents to update. To
+  specify an empty filter and match all ``Person`` instances, use an
+  empty ``Document`` object.
+- An update document that specifies the modifications. To view a list of
+  the available operators, see :manual:`Update Operators
+  </reference/operator/update/>` in the Server manual.
+
+The update methods return an ``UpdateResult`` type which provides
+information about the operation, including the number of documents
+modified by the update.
+
+Update a Single Person
+~~~~~~~~~~~~~~~~~~~~~~
+
+To update a single ``Person``, use the ``updateOne()`` method.
+
+The following example updates a ``Person`` named ``"Ada Byron"`` by
+setting their age to ``23`` and name to ``Ada Lovelace``:
+
+.. code-block:: java
+
+   collection.updateOne(
+       eq("name", "Ada Byron"), 
+       combine(set("age", 23), set("name", "Ada Lovelace"))
+   ).subscribe(new OperationSubscriber<>());
+
+Update Multiple Person Instances
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To update all ``Person``  instances that match a filter, use the
+``updateMany()`` method.
+
+The following example sets the ``zip`` field to ``null`` for all documents
+that have a ``zip`` value:
+
+.. code-block:: java
+
+   collection.updateMany(not(eq("zip", null)), set("zip", null))
+            .subscribe(new OperationSubscriber<>());
+
+Replace a Single Person
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Another method to change an existing ``Person`` instance is to use
+the ``replaceOne()`` method.
+
+The following example replaces a ``Person`` named ``"Ada Lovelace"``
+with the original ``Person`` instance:
+
+.. code-block:: java
+
+   collection.replaceOne(eq("name", "Ada Lovelace"), ada)
+       .subscribe(new OperationSubscriber<>());
+
+Delete Documents
+----------------
+
+To delete documents from a collection, you can use the collection's ``deleteOne()`` and ``deleteMany()`` methods.
+
+Pass a filter object to match the document or
+documents to delete. To specify an empty filter, use an empty
+``Document`` object.
+
+The delete methods return a ``DeleteResult`` type which provides
+information about the operation, including the number of documents
+deleted.
+
+Delete a Single Person That Matches a Filter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To delete a single ``Person`` that matches a filter, use the
+``deleteOne()`` method:
+
+The following example deletes one ``Person`` who lives in
+the city ``Wimborne``:
+
+.. code-block:: java
+
+   collection.deleteOne(eq("address.city", "Wimborne"))
+       .subscribe(new OperationSubscriber<>());
+
+Delete All Person Instances That Match a Filter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To delete multiple ``Person`` instances that match a filter, use the
+``deleteMany()`` method.
+
+The following example deletes all ``Person`` instances that live in
+the city ``London``:
```

Comment:
S: (applies to other sections as well) I think this syntax is more clear than using verbs like "live"
```suggestion
The following example deletes all ``Person`` instances that have a 
``city`` value of ``London``:
```